### PR TITLE
Updating Meredith Fuchs bio and images

### PR DESCRIPTION
**Pull request by sebworks**
Updating Meredith Fuchs bio and images.

## Changes
- Updated `src/the-bureau/about-deputy-director/index.html` to update bio copy.

## Screenshot
- 
screen shot 2015-09-08 at 9 08 47 am

## Testing
- Visit `http://localhost:7000/the-bureau/about-deputy-director/`

## Review
@anselmbradford 
@jimmynotjim
@KimberlyMunoz 
@duelj

### CODE REVIEW

File: src/the-bureau/about-deputy-director/index.html [hunk 1]
```diff
@@ -73,8 +68,8 @@ <h1 class="h4 u-mb5">
                         <ul class="list list__links">
                             {# TODO: Add a high-res photo when available. #}
```
Comment (anselmbradford) [inline src/the-bureau/about-deputy-director/index.html]: Remove `TODO`

**Comment by anselmbradford** [comment]
Just remove the TODO that has been addressed, otherwise :+1: Meredith looks happier with this change :smile:

**Comment by jimmynotjim** [comment]
:+1:

**Comment by anselmbradford** [comment]
:+1:

**Pull request merged**